back/engines/free/email_campaigns/config/locales/en.yml

```diff
@@ -397,11 +396,6 @@ en:
       header_title: 'There''s an update on "%{input_title}"'
       subject: 'Official feedback was posted on "%{input_title}"'
       preheader: 'There''s an update on an input you follow'
-    official_feedback_on_initiative_you_follow:
```

Comment:
Note: the related `official_feedback_on_idea_you_follow:` translations are not aligned between `en.yml` and `mailers.en.yml`, though I think many other translations have likewise drifted out of sync between those 2 files.

Reply:
It's not ideal but I think it's fine. It's normal that the translation files of the specs become more outdated as translations get changed. If we would use the real translations instead, we could get failing tests when translators make changes and this would either result in more complicated merges for Crowdin PRs or break the pipeline of master, which are all undesired situations. That's why we have separate translations for specs.

It's probably not a big problem if the specs are using slightly different translations, but we probably want to make sure that they use the same substitution variables (we don't test this because that's probably a bit complicated, but would be nice).